doc/man3/OSSL_STORE_INFO.pod

```diff
@@ -122,6 +122,14 @@ given to L<OSSL_STORE_open_file(3)>, the returned name will be the path of
 each object, so if C</foo/bar> was given and that path has the file
 C<cookie.pem>, the name C</foo/bar/cookie.pem> will be returned.
 
+The returned URI is considered canonical and must be unique and permanent
+for the storage where the object (or collection of objects if that's what
+each name represents) resides.
+It is up to each loader to ensure this.
```

Comment:
I'd rather be more explicit, as "Each loader is responsible for ensuring that it only returns unique and permanent URIs."